test/helper.rb

```diff
@@ -89,6 +89,20 @@ def table_has_column?(table, column)
     !ActiveRecord::Base.connection.select_values("desc #{table}").grep(column).empty?
   end
 
+  def migrator(direction = :up, path = 'migrations', target_version = nil)
+    migration_path = File.join(File.dirname(__FILE__), "/", path)
+    if ActiveRecord::VERSION::MAJOR >= 4
+      migrations = if ActiveRecord::VERSION::STRING >= "5.2.0"
+                     ActiveRecord::MigrationContext.new(migration_path).migrations
+                   else
+                     ActiveRecord::Migrator.migrations(migration_path)
+                   end
+      ActiveRecord::Migrator.new(direction, migrations, target_version)
+    else
+      ActiveRecord::Migrator.new(direction, migration_path, target_version)
+    end
```

Comment:
Would it be too much duplication with e.g.
```rb
    if ActiveRecord::VERSION::STRING >= "5.2.0"
      migrations = ActiveRecord::MigrationContext.new(migration_path).migrations
      ActiveRecord::Migrator.new(direction, migrations, target_version)
    elsif ActiveRecord::VERSION::MAJOR >= 4
      migrations = ActiveRecord::Migrator.migrations(migration_path)
      ActiveRecord::Migrator.new(direction, migrations, target_version)
    else
      ActiveRecord::Migrator.new(direction, migration_path, target_version)
    end
```

Reply:
Rubocop suggested me this style. Build fails with your code.

Reply:
Really? I guess I should go fix RuboCop then :-D

Reply:
I can use your code snippet if you would suggest me the way to ignore rubocop rule for this code.

Reply:
I don’t see RuboCop complaining on my code. What error are you seeing, exactly?